src/store/localStorage.js

```diff
@@ -0,0 +1,26 @@
+function extractDataToSave(state) {
+  return {
+    settings: { ...state.settings },
+  };
+}
+
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem('state');
+    if (serializedState === null) {
+      return undefined;
+    }
+    return JSON.parse(serializedState);
+  } catch (err) {
+    return undefined;
+  }
+};
+
+export const saveState = state => {
+  try {
+    const serializedState = JSON.stringify(extractDataToSave(state));
+    localStorage.setItem('state', serializedState);
```

Comment:
I think local storage is fine for now. I'd much prefer if this was saved on the sector object so that the player's can see the GM's preferences when they first load it up. Players could then always go in and modify this, but it wouldn't be synced locally or on the sector object.

Reply:
I first saved it to the sector object, but I didn't want to changes too many parts of the project because I am not very familiar with web development, react and firebase.

Reply:
Yeah, completely understandable! This is great